istioctl/pkg/install/verify.go

```diff
@@ -248,7 +247,7 @@ func NewVerifyCommand() *cobra.Command {
 		istioNamespace string
 		opts           clioptions.ControlPlaneOptions
 	)
-	verifyInstallCmd = &cobra.Command{
+	verifyInstallCmd := &cobra.Command{
```

Comment:
having a package level var was unnecessary and caused a panic when run in parallel